openpype/hosts/nuke/plugins/load/load_image.py

```diff
@@ -250,3 +252,17 @@ def remove(self, container):
 
         with viewer_update_and_undo_stop():
             nuke.delete(node)
+
+    def _get_node_name(self, representation):
```

Comment:
Since we're now implementing a function for this - shouldn't we also re-use it [on `load`](https://github.com/ynput/OpenPype/pull/5444/files#diff-1f8459817a04b371e8fb325f5768d8294c6754ee1cfaf893326ffdcc4cbab662R107-R116) to ensure the behavior stays the same?

Reply:
Make sense, I have aligned this behavior in the updated PR